src/main/java/org/jabref/logic/integrity/FieldCheckers.java

```diff
@@ -30,7 +30,15 @@ private static Multimap<Field, ValueChecker> getAllMap(BibDatabaseContext databa
         for (Field field : FieldFactory.getPersonNameFields()) {
             fieldCheckers.put(field, new PersonNamesChecker(databaseContext));
         }
+        fieldCheckers.put(StandardField.BOOKTITLE, new AuthorsContainedChecker());
+        fieldCheckers.put(StandardField.BOOKTITLE, new DateContainedChecker());
+        fieldCheckers.put(StandardField.BOOKTITLE, new PageNumbersContainedChecker());
+        fieldCheckers.put(StandardField.BOOKTITLE, new YearContainedChecker());
         fieldCheckers.put(StandardField.BOOKTITLE, new BooktitleChecker());
+        fieldCheckers.put(StandardField.JOURNALTITLE, new AuthorsContainedChecker());
+        fieldCheckers.put(StandardField.JOURNALTITLE, new DateContainedChecker());
+        fieldCheckers.put(StandardField.JOURNALTITLE, new PageNumbersContainedChecker());
```

Comment:
Journaltitle is a biblatex field, this needs to be moved down to the check isBiblatexMode
In BibTeX it's  just journal

Reply:
`JOURNAL` should be here - and below then additionally for `JOURNALTITLE`